habitat_sim/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 import builtins
 
-__version__ = "0.1.6"
+__version__ = "0.1.7"
```

Comment:
The conda version also need to be bumped, no? @mathfac Do we have a document on how to do releases?

Reply:
conda version is taken from this value.